Resources/Prototypes/SimpleStation14/Entities/Mobs/Customization/ears.yml

```
@@ -2,7 +2,7 @@
   id: EarsAvali
   bodyPart: HeadTop
   markingCategory: HeadTop
-  speciesRestriction: [Reptilian]
+  speciesRestriction: [Reptilian, Shadekin]
```

```suggestion
  speciesRestriction: [SlimePerson, Reptilian,]
```
Same as below. If I remember correctly, these are axolotl ears? They should look fine on Slime, right? And make up for the lack of Shadowkin ears.